### PR TITLE
simple-websocket-server: new port version 2.0.0-rc4

### PR DESCRIPTION
**Pull request by repagh**
, a c++, header-only, websocket development library

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x ] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mf2k** [comment]
It looks like this got add to PR #3800.

**Comment by repagh** [comment]
simple-websock-server is for ws / wss, simple-web-server is for http / https

**Comment by repagh** [comment]
Looks like you are right, I must have forgotten to switch branches somewhere....

**Comment by mf2k** [comment]
I realize that. What I mean is that you added 2 Portfiles in #3800. You added this one to that PR! So either close this PR or remove the Portfile from #3800.

**Comment by repagh** [comment]
I removed the portfile from #3800

File: devel/simple-websocket-server/Portfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.1
+
+name	  	    simple-websocket-server
+version		    2.0.0-rc4
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen @repagh openmaintainer
```
Comment (mf2k) [inline devel/simple-websocket-server/Portfile]: Same as for the other port. Use `{` and `}` around your email and GitHub handle to associate them.

**Comment by mf2k** [comment]
One minor fix here and then squash your commits into one with the message:

`simple-websocket-server: new port
`

File: devel/simple-websocket-server/Portfile (hunk 1 above)
Comment (repagh) [reply devel/simple-websocket-server/Portfile]: done

**Pull request merged**